libs/components/src/lib/menu/menu.ts

```diff
@@ -252,36 +250,32 @@ export class Menu extends Anchored(DelegatesAria(VividElement)) {
 		return this.isMenuItemElement(el);
 	};
 
-	private setFocus(focusIndex: number, adjustment: number): void {
+	private setFocus(focusIndex: number): void {
 		if (this.menuItems === undefined) {
 			return;
 		}
 
 		while (focusIndex >= 0 && focusIndex < this.menuItems.length) {
```

Comment:
This shouldn't be a loop anymore, right?
```suggestion
		if (focusIndex >= 0 && focusIndex < this.menuItems.length) {
```

Reply:
> This shouldn't be a loop anymore, right?

Yes, you're right. Updated in the next commit